examples/qaoa.lisp

```diff
@@ -67,9 +67,10 @@ GRAPH should be a list of edges, each represented as a pair (A B) of integer ver
 
 (defun cut-weight (graph cut)
   "Given a graph GRAPH and a cut (a list of vertices), compute the cut weight."
-  (loop :for (from to) :in graph
-        :sum (logxor (ldb (byte 1 from) cut)
-                     (ldb (byte 1 to) cut))))
+  (let ((val (loop :for (from to) :in graph
```

Comment:
I don't see the rationale for wrapping the accumulated value of the loop in a `let` form only to return it immediately.

Reply:
I should've paid more attention to what I was committing.